docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -3,324 +3,778 @@ This guide is maintained in the main Quarkus repository
+= Using simplified Hibernate ORM with Panache
+
 
 include::./attributes.adoc[]
 :config-file: application.properties
+//:COMM:
+:ENT:
 
-Hibernate ORM is the de facto JPA implementation and offers you the full breadth of an Object Relational Mapper.
-It makes complex mappings possible, but it does not make simple and common mappings trivial.
-Hibernate ORM with Panache focuses on making your entities trivial and fun to write in Quarkus.
+ifdef::ENT[]
+As a Java developer, you can use Hibernate ORM with Panache to simplify persistence with your {ProductName} applications.
+endif::ENT[]
 
-== First: an example
+ifdef::COMM[]
+You can use Hibernate ORM with Panache to simplify persistence with your {ProductName} applications.
+endif::COMM[]
 
-What we're doing in Panache is allow you to write your Hibernate ORM entities like this:
 
-[source,java]
-----
-@Entity
-public class Person extends PanacheEntity {
-    public String name;
-    public LocalDate birth;
-    public Status status;
+.Prerequisites
+* You have a {ProductName} Maven project configured with a Java database connectivy (JDBC) datasource and Hibernate ORM.
++
+NOTE: If you not define a JDBC driver, Hibernate ORM with Panache will not work.
 
-    public static Person findByName(String name){
-        return find("name", name).firstResult();
-    }
+ifdef::ENT[]
+* For information about creating a {ProductName} Maven project, see https://access.redhat.com/documentation/en-us///red_hat_build_of_quarkus/{ProductVersion}/html-single/creating_quarkus_applications_with_apache_maven/[_Creating Quarkus applications with Apache Maven_].
+endif::ENT[]
 
-    public static List<Person> findAlive(){
-        return list("status", Status.Alive);
-    }
+ifdef::COMM[]
+* For information about creating a {ProductName} Maven project, see https://quarkus.io/guides/maven-tooling[_Quarkus - Building Applications with Maven_].
+endif::COMM[]
 
-    public static void deleteStefs(){
-        delete("name", "Stef");
-    }
-}
-----
+* For information about configuring your {ProductName} Maven project with a JDBC datasource, see https://quarkus.io/guides/datasource[_Quarkus - Configuring your datasources_].
+* For information about configuring your {ProductName} Maven project Hibernate ORM, see https://quarkus.io/guides/hibernate-orm[_Quarkus - Quarkus - Hibernate ORM and JPA_].
 
-You have noticed how much more compact and readable the code is?
-Does this look interesting? Read on!
+For an example of a completed {ProductName} Maven project that uses the Panache extension, see the https://github.com/quarkusio/quarkus-quickstarts/tree/master/hibernate-orm-panache-quickstart[Hibernate ORM Panache Quickstart].
 
-NOTE: the `list()` method might be surprising at first. It takes fragments of HQL (JP-QL) queries and contextualizes the rest. That makes for very concise but yet readable code.
 
-== Setting up and configuring Hibernate ORM with Panache
+== Hibernate ORM with the {ProductName} Panache JPA extension
```

Comment:
I've seen this "{ProductName} Panache JPA extension" term used in a number of places. The correct name of this extension is "Hibernate ORM with Panache", could you replace it everywhere please? I'm aware that the original document may not have been consistent about this, probably because a number of people have edited it over the years and the name of the extension changed early on too, but it's a fine occasion to fix this.